projects/hslayers/components/layer-manager/layer-manager.component.ts

```diff
@@ -61,6 +61,7 @@ export class HsLayerManagerComponent
   layerlistVisible: boolean;
   hovering: boolean;
   physicalLayerListEnabled = false;
+  isCesiumActive = false;
```

Comment:
I think this could be more reactive using merge and async pipe. Also a bit more readable than assigning value to the same variable from two subscribtions.
Something like
```
cesiumActive$ = merge(
  this.hsEventBusService.cesiumLoads.pipe(
    map(() => true) 
  ),
  this.hsEventBusService.mapLibraryChanges.pipe(
    map(lib => lib === 'cesium')
  )
);
```

Reply:
Wow, I bet you are right. This is a bit too advanced to me. I will try to implement it as you suggest. (I feel like I shall finally do the online course about Angular + rxjs, I have bought a year ago 😁 )

Reply:
@FilipLeitner please, see commit [369a94c](https://github.com/hslayers/hslayers-ng/pull/4849/commits/369a94cc8c5d3ad0a78381cb4adbe42cda76ca19). Perhaps you can find even more elegant way (without the need of BehaviorSubject).

Reply:
Made a small adjustments in [c4db18f](https://github.com/hslayers/hslayers-ng/pull/4849/commits/c4db18f9cc38fef391e1f29efa8739a1a18a7980).  Its basically the same just reduced the number of subscriptions in template

Reply:
Thanks!